packages/core/admin/admin/src/content-manager/components/RelationInput/RelationInput.js

```diff
@@ -204,6 +204,10 @@ const RelationInput = ({
     onSearch();
   };
 
+  useEffect(() => {
+    listRef.current.scrollToItem(relations.length, 'end');
+  }, [relations]);
```

Comment:
Do you think you could come up with a test for this? :)

I'm also wondering about, whether this means after "save" we'd always scroll to the end of the list, because the `relations` dependency changes?

Reply:
> I'm also wondering about, whether this means after "save" we'd always scroll to the end of the list, because the relations dependency changes?

that's a good point. I'll look into this.